distribution/pom.xml

```diff
@@ -83,6 +74,7 @@
 				<executions>
 					<execution>
 						<id>attach-javadoc</id>
+						<phase>install</phase>
```

Comment:
New commit defines a user property ${no-javadoc} which defaults to true, and uses that to set maven.javdoc.skip .  

To generate javadoc use mvn package -Dno-javadoc=false